src/ray/core_worker/lib/java/io_ray_runtime_task_NativeTaskSubmitter.cc

```diff
@@ -293,6 +293,16 @@ Java_io_ray_runtime_task_NativeTaskSubmitter_nativeRemovePlacementGroup(
   THROW_EXCEPTION_AND_RETURN_IF_NOT_OK(env, status, (void)0);
 }
 
+JNIEXPORT jboolean JNICALL
+Java_io_ray_runtime_task_NativeTaskSubmitter_nativeWaitPlacementGroupReady(
+    JNIEnv *env, jclass p, jbyteArray placement_group_id_bytes, jint timeout_ms) {
+  const auto placement_group_id =
+      JavaByteArrayToId<ray::PlacementGroupID>(env, placement_group_id_bytes);
+  auto status = ray::CoreWorkerProcess::GetCoreWorker().WaitPlacementGroupReady(
```

Comment:
throw an exception to Java if status is not ok.

Reply:
fixed